index.js

```diff
@@ -0,0 +1,236 @@
+const RELEVANT_FIELDS =
+  "tld,name,capital,flags,population,region,subregion,currencies,languages,borders";
+
+const getRelevantURL = (url) => `${url}?fields=${RELEVANT_FIELDS}`;
+
+const freeChildren = (
+  container = document.getElementById("countries-container")
+) => {
+  while (container.firstChild) {
```

Comment:
Instead of iterating and removing, why not just write this:
```javascript
const freeChildren = (container = document.getElementById("countries-container")) => {
    container.innerHTML = ''; 
};
```
?